docs/restful-api/services/token-services.md

```diff
@@ -235,6 +235,9 @@ Content-Type: application/x-www-form-urlencoded</p></td>
 - <i>tokenType=APIKEY</i> - specifies the scope of validity of the APIKey; the only licensee with the given number can be validated using this APIKey<br />
```

Comment:
For the type
```
- <i>SHOP</i> - shop token is used to redirect customer to the <a href="netlicensing-shop">NetLicensing Shop</a> ('licenseeNumber' is mandatory)<br />
```
The comment `('licenseeNumber' is mandatory)` is not relevant for the permanent kind. Since the details are described in `netlicensing-shop` anyway, I suggest removing that comment.

Reply:
deleted